pkg/util/round_precision_test.go

```diff
@@ -0,0 +1,57 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/stretchr/testify/assert"
+)
+
+func number(a interface{}) fixedpoint.Value {
```

Comment:
there is a testhelper package you can import:

```
import . "github.com/c9s/bbgo/pkg/testing/testhelper"

// then you call Number()
```

Reply:
Done.